lib/src/handlers/mock_handler.dart

```diff
@@ -16,7 +15,7 @@ class MockHandler {
   Future<ResponseBody> createMock(String fullPath, dynamic data) async {
```

Comment:
add params in comment:
`/// Creates and writes the mock data in [fullPath] inside a file as a JSON object [data].`

Reply:
შესწორებულია, ოდნავ გადავაკეთე, `readMock`-იც.